vnext/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -45,6 +45,7 @@ TEST_CLASS (RNTesterIntegrationTests) {
 #pragma region Prototype tests
 
   BEGIN_TEST_METHOD_ATTRIBUTE(Logging)
+  TEST_IGNORE()
```

Comment:
Double-check the hang is a test issue and not a real regression, then link/comment an IOU issue to investigate getting it turned back on.

Reply:
After additional investigation found the culprit of the test failure. Test ignore no longer needed.